src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+export interface PreparedTodo extends Todo {
```

Comment:
Minor, but PreparedTodo is not perfect name here

Something like `AssignedTodo` or `TodoWithUser` would be better